pkg/expression/builtin_test.go

```diff
@@ -161,6 +163,71 @@ func TestDisplayName(t *testing.T) {
 	require.Equal(t, "other_unknown_func", GetDisplayName("other_unknown_func"))
 }
 
+func TestBuiltinFuncCache(t *testing.T) {
```

Comment:
I think it's better to add some concurrent tests.

Reply:
A new test case `TestBuiltinFuncCacheConcurrency` added